--- a/config.def.h	2024-08-21 19:55:22.287535570 +0200
+++ b/config.def.h	2024-08-21 19:55:19.046502179 +0200
@@ -4,7 +4,7 @@
 const unsigned int interval = 1000;
 
 /* text to show if no value can be retrieved */
-static const char unknown_str[] = "n/a";
+static const char unknown_str[] = "yeeta";
 
 /* maximum output string length */
 #define MAXLEN 2048
@@ -65,5 +65,11 @@
  */
 static const struct arg args[] = {
 	/* function format          argument */
-	{ datetime, "%s",           "%F %T" },
+    { wifi_perc, " WiFi %s% | ",          "wlan0" },
+    { run_command, " Vol %s | ",          "wpctl get-volume @DEFAULT_AUDIO_SINK@ | grep -Eo '[1-9]{1,3}'"},
+    { cpu_perc, " CPU %s% | ",          NULL },
+    { ram_free, " RAM %s | ",            NULL },
+    { battery_perc, " Bat %s% | ",          "BAT0" },
+    { uptime, "Uptime %s | ",              NULL },
+	{ datetime, "%s ",           "%A %H:%M" },
 };
